### ISSUE TITLE
Add recipe for bobwarfare

### ISSUE THREAD

**Issue by stefanaonx** [stefanaonx/spidertron-extended#6]
Add recipe for bobwarfare
- Need new recipes for the spidertrons: immolator, ghost weaver, mk2, mk3
- Need new recipe for the 2 batteries for the ghost weaver

**Comment by stefanaonx** [comment]
Resar suggested the following for the recipes on spider
`immolator

100 mech-armor-plate
1 mech-frame
8 mech-leg
2 rtg
200 refined-concrete
1 storage-tank
1 flamethrower


ghost weaver

50 mech-armor-plate
1 mech-frame
8 mech-leg
3 rtg
50 bob-contruction-robot-5
5 raw-productivity-module-3
10 titanium-chest
2 laser-turret
50 radar




 MK2

1 spidertron
50 mecha-armor-plate
2 rtg
5 raw-speed-module-3
5 efficiency-module-3
5 beacon-2


MK3

1 spidertronmk2
75 mecha-armor-plate
2 rtg
1 satellite
10 raw-speed-module-3
10 efficiency-module-3
5 beacon-3`

still need new recipe for the batteries

**Comment by SGZ-Creations** [comment]
In the immolator where's the logic of having concrete ?

Also can say that there are up to 8 module levels as bob's has it's own research for modules.

will also say after looking that the price of your recipe is more expensive on some as u use the higher tier of the item. waht i mean is that u need to produce the parts too. 

also want to suggest using the parts instead of previous tier spider

**Comment by Resarium** [comment]
> 
> 
> In the immolator where's the logic of having concrete ?
> 

The idea was that the concrete would offer complete immunity to fire (Immolator is actually immune to fire ingame) while the extra exoskeletons would bring its speed back to normal from such extra weight

> Also can say that there are up to 8 module levels as bob's has it's own research for modules.
> 

As of now, the Spidertrons use the same modules in their recipes as vanilla

> will also say after looking that the price of your recipe is more expensive on some as u use the higher tier of the item. waht i mean is that u need to produce the parts too.
> 
> also want to suggest using the parts instead of previous tier spider

As long as more advanced spidertrons require previous research, yes we can change so every one of them requires parts instead of previous spidertron, at least for bob's warfare (which is what we kinda decided on)

**Comment by stefanaonx** [comment]
Okay you can leave the batteries aside now, since on the next patch i'm going to make them automatically being added / removed to the ghost weaver.

**Comment by stefanaonx** [comment]
Cannot add the RTG to the recipe because it's not part of bobwarfare mod, need to look more into what bobwarfare gives as parts and figure it out.

**Comment by SGZ-Creations** [comment]
Thanks for info. Tho, way just not add the other bob's mods as optional(s) specific for the metals & ores. Just to give u more from bob's to use when present also avoiding it being expensive than vanilla ?

**Comment by SGZ-Creations** [comment]
Could u also add a new comment on how recipes would look. Of course u can copy the one u have then just edit that in a new comment. Just want to see what u have chosen to change at your end after current discussion.

**Comment by stefanaonx** [comment]
Thanks for the reminder, just committed version 0.3.8 with the changes for bob's warfare, only added extra stuff for bob's warfare not the other bob's mods, we would need to look over each of the bob's mods and make a compatibility and change for each one, but for the moment this should do.

**Comment by SGZ-Creations** [comment]
possibly u can switch the concrete with heat shielding from bob's revamp.

**Comment by stefanaonx** [comment]
Good idea, i'll add it as a dependency when also using the bob's revamp mod to switch to the heat shielding instead of concrete.

**Comment by SGZ-Creations** [comment]
thanks

**Comment by SGZ-Creations** [comment]
Immolator concret swap fix. Any thing else is  up to u

doing this soon ?:
> possibly u can switch the concrete with heat shielding from bob's revamp.

I rcomend setting a pre requsite for the heat shield
![image](https://user-images.githubusercontent.com/59801736/154813991-4b688791-aeb8-4bd4-8b0f-e0fb6c102460.png)

Itme name u need is "heat-shield-tile"
![image](https://user-images.githubusercontent.com/59801736/154814069-0e84ba0e-2356-47e4-b8bf-21282abb3481.png)

Mod Optional is Bobs revamp

**Comment by stefanaonx** [comment]
We should still add those heat-shields at some point.

**Comment by SGZ-Creations** [comment]
Yea. Hope it happens soon. Thanks for answer.

**Comment by SGZ-Creations** [comment]
want to ask of the project has been abandoned. Starting to get worried that the mod stops working when 2.0 gets out in:
Q3-Q4 2024 unless delayed more.

**Comment by stefanaonx** [comment]
> want to ask of the project has been abandoned. Starting to get worried that the mod stops working when 2.0 gets out in: Q3-Q4 2024 unless delayed more.

Thanks for still taking interests in the mod. Both me and Resarium still watches over factorio updates but work and family takes priority over updates to the mod. If the mod requires any changes to work with version 2.0 or if game breaking bugs appear we will try to make time to fix any of those.

**Comment by SGZ-Creations** [comment]
I can help fix bob compatibility when you get the mod working again for 2.0

**Comment by SGZ-Creations** [comment]
I'm totally willing to help.
